ci/tasks/run-int.yml

```diff
@@ -0,0 +1,15 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source: {repository: everlag/fog-google}
```

Comment:
This refers to an uploaded docker image created using ci/docker-image/Dockerfile, can you confirm that `fog/fog-google` at dockerhub is populated with that image?